app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java

```diff
@@ -3,26 +3,55 @@
 import com.example.calculadoramvp.mvp.model.CalculatorModel;
 import com.example.calculadoramvp.mvp.view.CalculatorView;
 
+import android.widget.Toast;
+
+import com.example.calculadoramvp.R;
+
 public class CalculatorPresenter {
-    private CalculatorModel model;
-    private CalculatorView view;
+
+    CalculatorView view;
+    CalculatorModel model;
+
 
     public CalculatorPresenter(CalculatorModel model, CalculatorView view){
         this.model = model;
         this.view = view;
     }
 
-    public void onButtonPressed(){
-        //TODO
-        /*IF IS =
-        IN MODEL I REALIZE A COUNT
-        MODIFIED THE VIEW TO SHOW THE RESULT
-        IF IS NOT =
-        IN THE VIEW SHOW NUMBERS*/
+    public void onNumberPressed(String number){
+
+        if (model.getOperand_1()==""){
+            model.setOperand_1(number);
+            view.setVisor(number);
+        }
+        else if (model.getOperand_1() != "" && model.getOperator()=="") {
+            model.setOperand_1(model.getOperand_1() + number);
+            view.setVisor(model.getOperand_1());
+        }
+        else {
+            model.setOperand_2(model.getOperand_2() + number);
+            view.setVisor(model.getOperand_1() + model.getOperator() + model.getOperand_2());
+        }
     }
 
-    public void onResetButtonPressed(){
+    public void onOperatorPressed(String operator){
 
+        if (model.getOperand_1() != "" && model.getOperand_2()==""){
```

Comment:
Please, remove this hardcode into a SringUtil.

Reply:
done